src/components/DuosHeader.js

```diff
@@ -75,6 +191,19 @@ class DuosHeader extends Component {
       isDataOwner = currentUser.isDataOwner;
     }
 
+    const dropdownLinks = {
+      statistics: {
+        'Votes Statistics': {
+          isRendered: !(isDataOwner || isResearcher) || isAdmin,
```

Comment:
I would prefer to have explicit roles defined in the affirmative here instead of the negative. For example, if we add a new role, say, "isSigningOfficial" and we don't explicitly negate that role here, then that role will gain access to this link. Same feedback for reviewed cases.

Reply:
This was the pre-existing condition on the navbar, but it's not obvious who's limited to seeing it (why Admin but not DataOwner or Researcher? Can a Researcher be a DAC member? What happens then?, etc.)

Reply:
Also I need to remove the conditional on Votes Statistics, there's no conditional on that link right now.

Reply:
Statistics are admin only right now, which is why we should clean this up. At some point, DAC Chairs will have a version of this page to look at, but we don't have it yet.